### DISCUSSION TITLE
Doge bots

### DISCUSSION THREAD

**Discussion by aatz1** [benawad/dogehouse#1819] [Ideas]
if you want to ultrase the discord, you need to have an api for creating moderation bots or music bots for the dogehouse

**Comment by imadill** [comment]


**Reply by 8BitSLH** [reply]
They don't even share the full code and have really poor documentation. Reasoning for not sharing the full code is to prevent people from trolling, which is pretty true, but Ben Awad himself should do something about it in the future.

**Reply by SirTenzin** [reply]
dogesword has a full wiki and a todo list for contributors :)

[sirtenzin/dogesword](https://github.com/sirtenzin/dogesword/)
